TerrorConsole/Assets/_Main/Scripts/SaveSystem/SaveConfigurationData.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using UnityEngine;
+
+namespace TerrorConsole
+{
+    [Serializable]
+    public class SaveConfigurationData
+    {
+        public int _sfxVolume;
+        public int _musicVolume;
+        public int _lenguageIndex;
```

Comment:
Thes should be written as `PascalCase` since are public and part of a data structure.